docs/reference/run.md

```diff
@@ -901,6 +902,11 @@ If you do block IO in the two containers at the same time, by, for example:
 You'll find that the proportion of time is the same as the proportion of blkio
 weights of the two containers.
 
+The `--blkio-weight-device` flag can set the weighting of a particular device to a value between 10 to 1000. With this, override the default value of group weight as specified by blkio.weight.
+For example, the command below create one container and set weight of /dev/sda 
```

Comment:
This one will need to be re-done, but I'll defer to the docs team to pick at it.